conf/function_worker.yml

```diff
@@ -0,0 +1,46 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+workerId: standalone
```

Comment:
We should have comments for each key explaining the purpose of each setting. 

Ideally, each setting should also have the default value, such that commenting that same line would have the same effect.

Reply:
this file is currently copied from functions-module, which I don't change any behavior at this moment. I would keep this change as simple as for now since it is already touching quite a few places. we can clean up these configuration files after we merge the functions dist with pulsar/all.